### PR TITLE
Entrega do trabalho

### PR DESCRIPTION
**Pull request by RonaldoLopes**

### CODE REVIEW

**Comment by edysegura** [comment]
Olá @RonaldoLopes !

Você prefere receber os meus comentários em Inglês ou Português?

**Comment by RonaldoLopes** [comment]
Boa noite, não tenho preferência, meu inglês não é dos melhores mas eu me
viro se for preciso.

Em 14 de jun de 2017 12:05 AM, "Edy Segura" <notifications@github.com>
escreveu:

File: ronaldo-alves/1.I/index.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Sistemas de informação</title>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" type="text/css" href="css/style.css">
+</head>
+<body>
+   <h1>Sistemas de informação Home Page</h1>
+   <h2>Sistemas de informação</h2>
```
Comment (edysegura) [inline ronaldo-alves/1.I/index.html:10]: Is there any reason to have added a duplicated title for "Sistemas de informação"?

File: ronaldo-alves/1.I/index.html [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+   <h3>Atuação profissional</h3>
+   <p>O Profissional de Tecnologia da Informação tem a capacidade de compreender os problemas do mundo real,<br/>
+      aplicação dos conhecimentos adquiridos na solução de problemas, capacidade de análise sistêmica dos<br/>
+      problemas, capacidade de síntese e de realização de projetos e, finalmente, capacidade de avaliação<br/>
+      e julgamento. As habilidades pessoais desse profissional são: pensamento sistêmico, capacidade de<br/>
+      resolução de problemas, pensamento crítico, disciplina pessoal, persistência, curiosidade, capacidade<br/>
+      de auto aprendizado e aberto a mudanças. Habilidades Interpessoais: trabalho colaborativo, capacidade<br/>
+      de comunicação, capacidade para resolução conjunta de problemas. Conhecimentos Técnicos: abstração,<br/>
+      representação e organização da informação, uso de ferramentas computacionais para aplicação dos conhecimentos.</p>
+    <h4>Corpo docente</h4>
```
Comment (edysegura) [inline ronaldo-alves/1.I/index.html:22]: Is it a sub-title of "Atuação profissional"?
Ref.: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements

File: ronaldo-alves/1.I/index.html [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        <li>Edy Segura</li>
+        <li>Elton Barbosa</li>
+        <li>Felipe Andery</li>
+        <li>Márcio Emílio</li>
+        <li>Paulo Siécola</li>
+        <li>Phyllipe Lima</li>
+        <li>Roberto Rocha</li>
+        <li>Rodrigo Pimenta</li>
+    </ul>
+    <h4>Informações</h4>
```
Comment (edysegura) [inline ronaldo-alves/1.I/index.html:36]: Is it a sub-title of "Atuação profissional"?
Ref.: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements

File: ronaldo-alves/1.II/index.html [hunk 4]
```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
```
Comment (edysegura) [inline ronaldo-alves/1.II/index.html:2]: As this document is in portuguese language we need to update the attribute lang to "pt-br"

File: ronaldo-alves/1.II/index.html [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+        <li>Curso de especialização em C# básico <a href="http://www.softblue.com.br/site/curso/id/10/CURSO+DE+FUNDAMENTOS+DE+C_+BASICO+ON+LINE+CS10" target="#">www.softblue.com.br</a></li>
+        <li>Curso de especialização em C# avançado <a href="http://www.softblue.com.br/site/curso/id/13/CURSO+DE+C_+AVANCADO+ON+LINE+CS13" target="#">www.softblue.com.br</a></li>
+        <li>Curso de especialização em Java avançado com JavaFX <a href="http://www.softblue.com.br/site/curso/id/2/CURSO+DE+JAVA+AVANCADO+ON+LINE+JV02" target="#">www.softblue.com.br</a></li>
+        <li>Curso de especialização em Android <a href="http://www.softblue.com.br/site/curso/id/7/CURSO+DE+ANDROID:+CRIANDO+APLICATIVOS+PARA+SMARTPHONES+E+TABLETS+BASICO+AO+AVANCADO+ON+LINE+AD07" target="#">www.softblue.com.br</a></li>
+        <li>Formado em Sistema de informação pela Univás(Universidade do Vale do Sapucai)</li>
+        <li>Especialista em certificação de rede de cobre e fibra ótica conforme os padrões técnicos atuais</li>
+    </ul>
+    <h3>Experiência profissional</h3>
```
Comment (edysegura) [inline ronaldo-alves/1.II/index.html:27]: Is it a sub-title of your "Formação e qualificação"?
Ref.: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements

File: ronaldo-alves/1.III/index.html [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+        <p>Em 1914, ano em que começou a Primeira Guerra Mundial, Tolkien se tornou noivo de Edith Bratt. E, no ano seguinte, mesmo recebendo com honras o diploma em Literatura de Língua Inglesa, ele não conseguiu escapar da convocação militar, e em 1916, depois de casar-se com Edith Bratt, ele foi enviado à guerra. Conseguiu sobreviver à Batalha do Somme, em uma mal-sucedida campanha na França, onde mais de 500 mil soldados foram mortos. Em 1917 nasceu o seu primeiro filho, e no ano seguinte, J.R.R.Tolkien foi enviado de volta à Inglaterra, por ter contraído tifo. Foi nesse período de dispensa em que começou a escrever o Livro dos Contos Perdidos (The Book of Lost Tales), que mais tarde teria seu nome mudado para O Silmarillion, em 1919, quando Tolkien conseguiu retornar para Oxford.</p>
+    </section>
+    <section id="resenha">
+        <h3>Resenha do livro</h3>
```
Comment (edysegura) [inline ronaldo-alves/1.III/index.html:32]: Why did you choose to use `<h3>` instead of `<h2>` here?

File: ronaldo-alves/1.III/index.html [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+        <p>Quanto mais perto de Mordor (ou de Sauron) o Um anel chega, mais forte sua vontade fica. Frodo e Golum que estão intimamente ligados ao anel cada vez mais fraquejam diante de seus desejos de ficarem com ele e Sam começa a crescer como personagem na história querendo sempre o melhor para seu amigo, colocando-o sempre à frente de sí mesmo.</p>
+    </section>
+    <section id="chapter01">
+        <h4>Capítulo 1</h4>
```
Comment (edysegura) [inline ronaldo-alves/1.III/index.html:37]: Why did you choose to use `<h4>` instead of `<h2>` here?

File: ronaldo-alves/1.III/index.html [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+        <h5>Minas Tirith</h5>
```
Comment (edysegura) [inline ronaldo-alves/1.III/index.html:38]: Why did you choose to use `<h5>` instead of `<h3>` here?

File: ronaldo-alves/1.III/index.html [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+        <p>Houvera a primeira cavalgada, numa velocidade alucinante e sem paradas, e depois, na aurora, Pippin vislumbrara um pálido brilho dourado, e eles tinham chegado à silenciosa cidade e à grande casa vazia sobre a colina. E mal tinham alcançado esse abrigo quando a sombra alada passou sobrevoando mais uma vez, deixando os homens descorçoados de medo. Mas Gandalf lhe disse palavras suaves, e Pippin adormeceu num canto, cansado mas inquieto, percebendo vagamente as idas e vindas, e os homens conversando, e Gandalf dando ordens. E, depois, outra cavalgada, outra cavalgada na noite. Aquela era a segunda, não, a terceira noite desde que o hobbit olhara dentro da Pedra. E com essa lembrança hedionda despertou completamente, e tremeu, o ruído do vento se encheu de vozes ameaçadoras.</p>
+    </section>
+    <section id="chapter02">
+        <h4>Capitulo 2</h4>
```
Comment (edysegura) [inline ronaldo-alves/1.III/index.html:43]: Why did you choose to use `<h4>` instead of `<h2>` here?

File: ronaldo-alves/1.III/index.html [hunk 10]
```diff
@@ -0,0 +1,58 @@
...
+        <h5>A passagem da companhia cinzenta</h5>
```
Comment (edysegura) [inline ronaldo-alves/1.III/index.html:44]: Why did you choose to use `<h5>` instead of `<h3>` here?

File: ronaldo-alves/1.III/index.html [hunk 11]
```diff
@@ -0,0 +1,58 @@
...
+        <p>GANDALF FORA EMBORA, e o ruído surdo dos cascos de Scadufax se perdia na noite, quando Merry voltou ao encontro de Aragorn. Trazia apenas um embrulho pequeno, pois perdera sua mochila no Parth Galen, e tudo o que tinha eram algumas poucas coisas úteis que apanhara nas ruínas de Isengard. Flasufel já estava selado. Legolas e Gimli, com seu cavalo, estavam ali perto.</p>
+        <p>— Então quatro membros da Comitiva ainda restam — disse Aragorn — Vamos continuar cavalgando juntos. Mas não iremos sozinhos, como eu havia pensado. Agora o rei está determinado a partir imediatamente. Desde a passagem da sombra alada, ele deseja retornar ás colinas sob a proteção da noite.</p>
+    </section>
+    <section id="apendice">
+        <h4>ANAIS DOS REIS E GOVERNANTES</h4>
```
Comment (edysegura) [inline ronaldo-alves/1.III/index.html:49]: Why did you choose to use `<h4>` instead of `<h3>` here?

File: ronaldo-alves/1.III/index.html [hunk 12]
```diff
@@ -0,0 +1,58 @@
...
+        <p>Quanto às fontes da maior parte do conteúdo dos próximos Apêndices, especialmente do A até o D, ver nota no final do Prólogo. A Seção A III, O Povo de Durin, origina-se provavelmente de Gimli, o anão, que manteve sua amizade com Peregrin e Meriadoc, e visitou-os muitas vezes em Gondor e em Rohan.</p>
+        <p>As lendas, histórias e estudos que se encontram nas fontes são muito extensos. Destes, apenas seleções, muitas vezes bastante resumidas, são apresentadas aqui. Seu principal propósito é esclarecer a Guerra do Anel e suas origens, e preencher várias lacunas da história principal. As antigas lendas da Primeira Era, nas quais residia o principal interesse de Bilbo, são tratadas muito brevemente, uma vez que dizem respeito aos antepassados de Elrond e aos reis e lideres númenorianos.</p>
+    </section>
+    <section id="ref">
+        <h5>Referência bibliográfica</h5>
```
Comment (edysegura) [inline ronaldo-alves/1.III/index.html:54]: Why did you choose to use `<h5>` instead of `<h2>` here?

**Review by edysegura — COMMENTED** [review]
Hi @RonaldoLopes !

Please let me know if you have any doubts about my comments.

Cheers,

File: ronaldo-alves/1.I/index.html (hunk 1 above)
Comment (RonaldoLopes) [reply ronaldo-alves/1.I/index.html:10]: Nenhuma, falta de atenção nesse caso.

File: ronaldo-alves/1.II/index.html (hunk 4 above)
Comment (RonaldoLopes) [reply ronaldo-alves/1.II/index.html:2]: Thanks, I need to pay more attention.

**Comment by RonaldoLopes** [comment]
Tanto faz, não tenho problemas com inglês.

Em 9 de ago de 2017 12:59 AM, "Edy Segura" <notifications@github.com>
escreveu:

Merged #5 <https://github.com/inatel/t141-html-exercises/pull/5>.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/inatel/t141-html-exercises/pull/5#event-1198382919>,
or mute
the thread

**Pull request merged**